libraries/support/azure/rest.rb

```diff
@@ -5,6 +5,10 @@
 
 module Azure
   class Rest
+    USER_AGENT = 'User-Agent'
+    INSPEC_USER_AGENT = 'Chef-Inspec-pid-18d63047-6cdf-4f34-beed-62f01fc73fc2'
+    private_constant  :USER_AGENT, :INSPEC_USER_AGENT
```

Comment:
Strings are already frozen on line 1 https://github.com/inspec/inspec-azure/blob/e11b70eeb1546462f93afc126c27ed32a17b52d1/libraries/support/azure/rest.rb#L1 and these vars should be private by default. I don't think you need line 10.

Reply:
@dmccown I can't find any documentation indicating default scope of constants is private. I used [this](http://www.chrisrolle.com/en/blog/private-constants-in-ruby) and [this](https://stackoverflow.com/a/11885328) as reference. Is it a new ruby version change?

Reply:
I was wrong in assuming scope, thanks for pointing that out.